### PR TITLE
PR to fix Trac 1202 (Make Image and Scale osd types obey osd_configuration setting)

### PR DESCRIPTION
**Pull request by pohlinkzei**
This PR shall fix [trac-1202](http://trac.navit-project.org/ticket/1202).
After looking into it again after a year, I could fix the behavior for Image and Scale.

Additionally I renamed the OSD flags to what they probably do.

I took a new branch (with typo -.- ) because we do not want to see all commit from the past  year. 

#235 can stay closed.

### CODE REVIEW

File: navit/osd/core/osd_core.c [hunk 1]
```diff
@@ -3901,14 +3909,14 @@ osd_scale_draw(struct osd_priv_common *opc, struct navit *nav)
 
 	if (!navit_get_attr(nav, attr_transformation, &transformation, NULL))
 		return;
-	if (this->use_overlay) {
+	//if (this->use_overlay) {
```
Comment (pgrandin) [inline navit/osd/core/osd_core.c]: We should probably delete code instead of commenting it

File: navit/osd/core/osd_core.c [hunk 2]
```diff
@@ -3943,14 +3951,14 @@ osd_scale_draw(struct osd_priv_common *opc, struct navit *nav)
 	graphics_draw_rectangle(opc->osd_item.gr, opc->osd_item.graphic_fg, p+6, 4,opc->osd_item.h/5+4);
 	graphics_draw_rectangle(opc->osd_item.gr, opc->osd_item.graphic_fg, p+7, p[1].x-p[0].x, 4);
 	graphics_draw_rectangle(opc->osd_item.gr, opc->osd_item.graphic_fg, p+8, 4,opc->osd_item.h/5+4);
-	graphics_draw_lines(opc->osd_item.gr, this->black, p, 2);
-	graphics_draw_lines(opc->osd_item.gr, this->black, p+2, 2);
-	graphics_draw_lines(opc->osd_item.gr, this->black, p+4, 2);
+	graphics_draw_lines(opc->osd_item.gr, opc->osd_item.graphic_fg_text /*this->black*/, p, 2);
```
Comment (pgrandin) [inline navit/osd/core/osd_core.c]: Same here, it's probably not worth keeping the old code

**Comment by pgrandin** [comment]
It's nice to get another contribution from you @pohlinkzei!
Would you mind attaching a screenshot also showing the improvement of your PR?

Thanks!

File: navit/osd/core/osd_core.c (hunk 2 above)
Comment (pohlinkzei) [reply navit/osd/core/osd_core.c]: Ha! I obviously missed those two....

**Comment by pohlinkzei** [comment]
Yes, I Will.

**Comment by pohlinkzei** [comment]
Complete config file to be found here: https://github.com/pohlinkzei/Pi_MFD_Layout/tree/trac1202

**Comment by pgrandin** [comment]
Nice work

**Pull request merged**